src/App.js

```diff
@@ -16,6 +19,14 @@ const App = () => {
         console.log(err);
       });
   };
+  const fetchProductData = async () => {
```

Comment:
Incorrect use of async await! No point using .then and .catch when using async await.
Please learn about this!